webapp/src/components/github_repo_selector/github_repo_selector.jsx

```diff
@@ -30,25 +33,65 @@ export default class GithubRepoSelector extends PureComponent {
     }
 
     componentDidMount() {
-        this.props.actions.getRepos();
+        this.props.actions.getOrgs();
+        this.props.actions.getReposByOrg("");
     }
 
-    onChange = (_, name) => {
-        const repo = this.props.yourRepos.find((r) => r.full_name === name);
+    onChangeForOrg = (_, org) => {
+        if (this.state.org != org) {
+            this.setState({org: org}, () => {this.props.actions.getReposByOrg(org)});
```

Comment:
Does this code actually need to wait for `setState` to complete?

```js
this.setState({org});
this.props.actions.getReposByOrg(org)
```

Reply:
Not necessary, changed.